Algebraic Topology/algebraic-topology.tex

```diff
@@ -224,18 +225,18 @@ \section{Homotopy}
 \end{thm}
 \begin{thm}
   If $X$ is path connected, then for any $x_a, x_b \in X$,
-  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$
-  notation If $X$ is path connected, we can denote the fundamental
+  $$\Pi_1(X, x_a) \isom \Pi_1(X, x_b).$$ %
+  Notation: If $X$ is path connected, we can denote the fundamental
   group with $\pi_1(X)$ or $\pi_1 X$.
 \end{thm}
 \begin{example}
-  pretty picture
+  pretty picture %
```

Comment:
you are correct regarding the pictures, the original lacking pictures were from your commits were they not?